### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by matjack1** [omersiar/esp-rfid#661] [General]
I'm very happy to release v2.0.0.

Please find all the details of the changes in the [CHANGELOG](https://github.com/esprfid/esp-rfid/blob/stable/CHANGELOG.md).

The upgrade from v1 should be possible, if you have problems please open an issue.

The most important breaking changes are on the MQTT API, you can find the documentation in the [README-MQTT](https://github.com/esprfid/esp-rfid/blob/stable/README-MQTT.md) file.

<hr /><em>This discussion was created from the release <a href='https://github.com/esprfid/esp-rfid/releases/tag/2.0.0'>v2.0.0</a>.</em>

**Comment by arsjoni** [comment]
Your
this version is much more regular and when you download the data it is more fixed
please, can you explain to me how to do MQTT Settings, I don't understand them very well
